README.md

```
@@ -84,150 +89,106 @@ tsumugai は以下を **一切行いません**。
+- `q` で終了
 
-    Ok(())
-}
-```
+ゲーム実装前にシナリオを体験できます。
+
+---
 
-### 従来API（互換性維持）
+## 想定している利用者
+	•	ノベルゲームを コードで管理したいエンジニア
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)